### PR TITLE
Add overload method to enable CloudWatch Sink through config file.

### PR DESCRIPTION
**Pull request by hzhengran**
`ReadFrom.AppSettings()` can only accept `string` input. So add a overload method to accept `string` params.

### CODE REVIEW

File: src/Serilog.Sinks.AwsCloudWatch/AwsCloudWatchConfigurationExtension.cs [hunk 1]
```diff
@@ -29,5 +29,29 @@ public static LoggerConfiguration AmazonCloudWatch(this LoggerSinkConfiguration
...
+        {
+            // validating input parameters
+            if (loggerConfiguration == null) throw new ArgumentNullException(nameof(loggerConfiguration));
+            if (string.IsNullOrEmpty(logGroupName)) throw new ArgumentException(nameof(logGroupName));
+            if (string.IsNullOrEmpty(accessKey)) throw new ArgumentException(nameof(accessKey));
+            if (string.IsNullOrEmpty(secretKey)) throw new ArgumentException(nameof(secretKey));
+            if (string.IsNullOrEmpty(region)) throw new ArgumentException(nameof(region));
+
+            CloudWatchSinkOptions options = new CloudWatchSinkOptions() { LogGroupName = logGroupName };
+            IAmazonCloudWatchLogs client = new AmazonCloudWatchLogsClient(accessKey, secretKey, Amazon.RegionEndpoint.GetBySystemName(region));
```
Comment (wparad) [inline src/Serilog.Sinks.AwsCloudWatch/AwsCloudWatchConfigurationExtension.cs]: This forces an implementation of a IAmazonCloudWatchLogs client, which shouldn't be `newed` up here.

File: src/Serilog.Sinks.AwsCloudWatch/AwsCloudWatchConfigurationExtension.cs [hunk 2]
```diff
@@ -29,5 +29,29 @@ public static LoggerConfiguration AmazonCloudWatch(this LoggerSinkConfiguration
...
+        /// <summary>
+        /// Activates logging to AWS CloudWatch
+        /// </summary>
+        /// <param name="loggerConfiguration">The LoggerSinkConfiguration to register this sink with.</param>
+        /// <param name="logGroupName">The log group name you want to use.</param>
+        /// <param name="accessKey">AWS Access Key ID.</param>
+        /// <param name="secretKey">AWS Secret Access Key.</param>
+        /// <param name="region">The region to connect.</param>
+        /// <returns></returns>
+        public static LoggerConfiguration AmazonCloudWatch(this LoggerSinkConfiguration loggerConfiguration, string logGroupName, string accessKey, string secretKey, string region)
```
Comment (wparad) [inline src/Serilog.Sinks.AwsCloudWatch/AwsCloudWatchConfigurationExtension.cs]: Please use a class to contain all the options we care about passing in, the way this currently is does not make it scalable.

**Comment by hzhengran** [comment]
The config file is mostly the same as #28 :

```
<appSettings>    
    <add key="serilog:minimum-level" value="Verbose" />
    <add key="serilog:using:AmazonCloudWatch" value="Serilog.Sinks.AwsCloudWatch" />
    <add key="serilog:write-to:AmazonCloudWatch.logGroupName" value="***" />
    <add key="serilog:write-to:AmazonCloudWatch.accessKey" value="***" />
    <add key="serilog:write-to:AmazonCloudWatch.secretKey" value="***" />
    <add key="serilog:write-to:AmazonCloudWatch.region" value="us-east-1" />
</appSettings>
```
And the code is:

```
Log.Logger = new LoggerConfiguration()
                .ReadFrom.AppSettings()
                .CreateLogger();
```

**Comment by wparad** [comment]
I may be missing something but could you help me figure out how using this method would allow setting the `LogEventRenderer` as part of the `CloudWatchOptions` configuration.  This is an important feature we are offering, and it isn't clear from the [Serilog documentation](https://github.com/serilog/serilog/wiki/AppSettings) on how to do this.  (I ask because the constructor we currently have wouldn't support this as is, and I don't want to diverge from the functionality that is currently being provided by the other constructor).

**Comment by wparad** [comment]
Is this relevant? https://github.com/serilog/serilog-settings-configuration/issues/53

**Comment by hzhengran** [comment]
Another PR #32 covered this.

**Pull request closed without merging**